components/dashboard/src/components/Modal.tsx

```diff
@@ -213,6 +213,16 @@ export const ModalFooter: FC<ModalFooterProps> = ({ className, alert, children }
     );
 };
 
+export const ModalBaseFooter: FC<{ className?: string; children: ReactNode }> = ({ className, children }) => {
```

Comment:
I don't really understand the naming here. What does "base" mean in this context? Could we just make the existing modal footer customizable so that you could provide custom `className`s?

Reply:
You could try it in preview env with start-local, it will make things harder and may affect other views

Reply:
Let's leave as-is for now 🙏